packages/infolists/resources/views/components/text-entry.blade.php

```diff
@@ -193,8 +203,24 @@ class="cursor-pointer max-w-max"
                     @if ($limitedArrayStateCount = count($limitedArrayState ?? []))
                         <{{ $isListWithLineBreaks ? 'li' : 'div' }}
                             class="text-sm text-gray-500 dark:text-gray-400"
+                            x-on:click.prevent="isLimited = ! isLimited"
```

Comment:
We don't need these attributes on this wrapper anymore, right?

Reply:
Which attributes?  The classes?

Reply:
And the click handler. I guess these changes are from before we moved to using links?

Reply:
Done